prime-router/docs/releases/2021-11-04-release-notes.md

```diff
@@ -0,0 +1,26 @@
+#  ReportStream Release Notes
+
+*November 4, 2021*
+
+## General useful links:
+
+- All Schemas are documented here:  [Link to detailed schema dictionaries](../schema_documentation)
+- The Hub API is documented here: [Hub OpenApi Spec](../openapi.yml)
+- [Click here for all release notes](../releases)
+
+## For this release
+
+### Updates to the Warnings and Errors JSON response for `/api/waters`
+
+Building upon the error and warning message updates from the 2021-10-19 release, a few changes have been made to
```

Comment:
These release notes are really well written - thanks!     Could you just add an example of what the new release notes look like?

Reply:
Added! 👍🏼